be/src/exec/vectorized/json_scanner.cpp

```diff
@@ -75,8 +75,12 @@ StatusOr<ChunkPtr> JsonScanner::get_next() {
         _cur_file_eof = false;
     }
     Status status = _cur_file_reader->read_chunk(src_chunk.get(), _max_chunk_size, _src_slot_descriptors);
-    if (status.is_end_of_file()) {
-        _cur_file_eof = true;
+    if (!status.ok()) {
+        if (status.is_end_of_file()) {
+            _cur_file_eof = true;
+        } else {
+            return status;
```

Comment:
If the status is an error, will it not be paused?

Reply:
No. The job would be aborted and retry later.